### ISSUE TITLE
Uncaught error in HA logs

### ISSUE THREAD

**Issue by hvorragend** [MelleD/lovelace-expander-card#129]
**Checklist:**

- [X] I updated to the latest version available
- [X] I cleared the cache of my browser

Each time I open a dashboard with the expanded-card, I found the following error in my ha-logs:

```
Logger: frontend.js.latest.202406050
Quelle: components/system_log/__init__.py:333
Erstmals aufgetreten: 21:28:55 (3 Vorkommnisse)
Zuletzt protokolliert: 21:28:55

Uncaught error from Mobile Safari 17.5 on iOS 17.5.1 TypeError: undefined is not an object (evaluating 'd.addEventListener') /hacsfiles/lovelace-expander-card/expander-card.js:1:40900 map ([native code]::) connectedCallback (/hacsfiles/lovelace-expander-card/expander-card.js:1:8480) this (src/scoped-custom-element-registry.js:248:47) appendChild ([native code]::) _addCardToColumn (src/panels/lovelace/views/hui-masonry-view.ts:255:27) Promise.all (src/panels/lovelace/views/hui-masonry-view.ts:237:8)
```

This happens with all browsers. Not just from smartphones.
The contents of the expander card do not seem to play a role either.

**Comment by MelleD** [comment]
Can you send me an example (yaml)?

EDIT: Found it, you don't use a title card, right?

**Comment by MelleD** [comment]
FIxed in 1.2.8

**Comment by hvorragend** [comment]
Perfect. The error is gone.
